addons/settings/fnc_gui_createCategory.sqf

```diff
@@ -186,7 +172,7 @@ private _lastSubCategory = "$START";
         };
 
         // ----- set setting name
-        private _ctrlSettingName = _ctrlSettingGroup controlsGroupCtrl IDC_SETTING_NAME;
+        private _ctrlSettingName = GET_CTRL_NAME(_ctrlSettingGroup);
```

Comment:
I do not like these. Seems like pointless indirection. The statement returning a "ctrl name" is already document by the identifier of the assignment.

Reply:
It was for the sake of reducing text, I found it to be an eyesore.